eras/conway/test-suite/cardano-ledger-conway-test.cabal

```
@@ -30,19 +30,19 @@ library
 
     build-depends:
         base >=4.14 && <4.19,
-        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.5.1,
+        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} ^>=1.6,
         cardano-ledger-alonzo-test,
-        cardano-ledger-babbage >=1.3 && <1.7,
-        cardano-ledger-babbage-test,
-        cardano-ledger-binary >=1.0,
+        cardano-ledger-babbage ^>=1.6,
+        cardano-ledger-babbage-test ^>=1.1.1.9,
+        cardano-ledger-binary ^>=1.3,
         cardano-strict-containers,
-        cardano-ledger-conway:{cardano-ledger-conway, testlib} >=1.9 && <1.13,
-        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.8.1,
-        cardano-ledger-allegra ^>=1.2,
-        cardano-ledger-mary ^>=1.4,
-        cardano-ledger-shelley-ma-test >=1.1,
-        cardano-ledger-shelley-test >=1.1,
-        cardano-ledger-shelley >=1.6,
+        cardano-ledger-conway:{cardano-ledger-conway, testlib} ^>=1.12,
+        cardano-ledger-core:{cardano-ledger-core, testlib} ^>=1.10,
+        cardano-ledger-allegra ^>=1.3,
+        cardano-ledger-mary ^>=1.5,
+        cardano-ledger-shelley-ma-test ^>=1.2.1.6,
+        cardano-ledger-shelley-test ^>=1.3.0.1,
+        cardano-ledger-shelley ^>=1.9,
```

Same as in my other comment, nothing has changed that warrants a lower and upper bounds
```suggestion
        cardano-ledger-conway:{cardano-ledger-conway, testlib} >=1.9 && <1.13,
        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.8.1,
        cardano-ledger-allegra >=1.2,
        cardano-ledger-mary >=1.4,
        cardano-ledger-shelley-ma-test >=1.1,
        cardano-ledger-shelley-test >=1.1,
        cardano-ledger-shelley >=1.6,
```